bigtable/google/cloud/bigtable/row_set.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""User-friendly container for Google Cloud Bigtable RowSet """
+
+
+from google.cloud._helpers import _to_bytes
+
+
+class RowSet(object):
+    """ Convenience wrapper of google.bigtable.v2.RowSet
```

Comment:
Docstrings should have a single line summary, with expanded comments following a blank line, and dedented.  E.g.:

```python
class RowSet(object):
    """ Convenience wrapper of google.bigtable.v2.RowSet

    Useful for creating a set of row keys and row ranges, which can
    be passed to yield_rows method of class:`.Table.yield_rows`.
    """
```

Reply:
Done, thanks.